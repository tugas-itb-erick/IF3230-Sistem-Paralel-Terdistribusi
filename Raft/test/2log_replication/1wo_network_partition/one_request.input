3
set_timeout 1 0
simulate 10
client_send_request 1 + 5
client_send_request 1 * 5
client_send_request 1 - 3
simulate 100
get_state 1
get_state 2
get_state 3
end_simulation
