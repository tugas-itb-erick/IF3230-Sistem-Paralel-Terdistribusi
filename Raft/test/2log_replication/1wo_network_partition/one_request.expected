S1 leader term:2 voted_for:1 commit_index:3 data:22 logs:T0[0] T2[+5*5-3]
S2 follower term:2 voted_for:1 commit_index:3 data:22 logs:T0[0] T2[+5*5-3]
S3 follower term:2 voted_for:1 commit_index:3 data:22 logs:T0[0] T2[+5*5-3]
